aws-test/tests/aws_macie2_classification_job/variables.tf

```diff
@@ -0,0 +1,97 @@
+
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20200155-create-update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "aws_profile" {
+  type        = string
+  default     = "default"
+  description = "AWS credentials profile used for the test. Default is to use the default profile."
+}
+
+variable "aws_region" {
+  type        = string
+  default     = "us-east-1"
+  description = "AWS region used for the test. Does not work with default region in config, so must be defined here."
+}
+
+variable "aws_region_alternate" {
+  type        = string
+  default     = "us-east-2"
+  description = "Alternate AWS region used for tests that require two regions (e.g. DynamoDB global tables)."
+}
+
+provider "aws" {
+  profile = var.aws_profile
+  region  = var.aws_region
+}
+
+provider "aws" {
+  alias   = "alternate"
+  profile = var.aws_profile
+  region  = var.aws_region_alternate
+}
+
+data "aws_partition" "current" {}
+data "aws_caller_identity" "current" {}
+data "aws_region" "primary" {}
+data "aws_region" "alternate" {
+  provider = aws.alternate
+}
+
+data "null_data_source" "resource" {
+  inputs = {
+    scope = "arn:${data.aws_partition.current.partition}:::${data.aws_caller_identity.current.account_id}"
+  }
+}
+
+# resource "aws_s3_bucket" "test" {
+#   bucket = var.resource_name
+# }
+
+resource "aws_macie2_account" "test" {
+  finding_publishing_frequency = "FIFTEEN_MINUTES"
+  status                       = "ENABLED"
+}
+
+resource "aws_macie2_classification_job" "named_test_resource" {
+  depends_on = [aws_macie2_account.test]
+  job_type = "ONE_TIME"
+  name     = var.resource_name
+  s3_job_definition {
+    bucket_definitions {
+      account_id = "986325076436"
+      buckets = ["appstream-app-settings-us-east-1-986325076436-vz8cc0a5"]
```

Comment:
Can we remove the reference of existing bucket, and create a new bucket for test?